### ISSUE TITLE
Whitespace sometimes discarded

### ISSUE THREAD

**Issue by MaxBarraclough** [ArthurClemens/mithril-template-converter#5]
When converting the following HTML fragment, the converter discards the whitespace between the two words.

```HTML
<p>hello <a href="https://example.com/">world</a></p>
```
	
Is converted to:
```JavaScript
m("p",
  [
    "hello",
    m("a", {"href":"https://example.com/"}, 
      "world"
    )
  ]
)
```

**Comment by ArthurClemens** [comment]
What is the expected output?

**Comment by MaxBarraclough** [comment]
I'd expect to see `"hello "`, rather than `"hello"`, preserving the space separating the words.

**Comment by ArthurClemens** [comment]
Simply omitting `trim()` doesn't cut it, because newlines still need to be removed.

A workable compromise is:
* Don't trim
* Remove newlines and tabs

Then this source:
```html
<p>
  hello  <a href="https://example.com/">world</a>
</p>
```

Is converted to:

```javascript
m('p',
  [
    ' hello ',
    m('a', {'href':'https://example.com/'}, 
      'world'
    )
  ]
)
```

Note:
* Multiple surrounding spaces are converted to single spaces
* The spaces between `<p>` and `hello` are converted to a single space

**Comment by ArthurClemens** [comment]
Implemented in 1.3.0.
